source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -32,24 +33,39 @@ def __init__(self, obj, position, _rangeObj=None):
 
 
 class winConsoleUIA(Terminal):
+	STABILIZE_DELAY = 0.03
 	_TextInfo = consoleUIATextInfo
 	_isTyping = False
 	_lastCharTime = 0
 	_TYPING_TIMEOUT = 1
 
 	def _reportNewText(self, line):
 		# Additional typed character filtering beyond that in LiveText
-		if self._isTyping and time.time() - self._lastCharTime <= self._TYPING_TIMEOUT:
+		if (
+			self._isTyping
+			and time.time() - self._lastCharTime <= self._TYPING_TIMEOUT
+		):
 			return
 		super(winConsoleUIA, self)._reportNewText(line)
 
 	def event_typedCharacter(self, ch):
 		if not ch.isspace():
 			self._isTyping = True
+		if ch in ('\r', '\t'):
+			# Clear the typed word buffer for tab and return.
+			# This will need to be changed once #8110 is merged.
+			speech.curWordChars = []
 		self._lastCharTime = time.time()
 		super(winConsoleUIA, self).event_typedCharacter(ch)
 
-	@script(gestures=["kb:enter", "kb:numpadEnter", "kb:tab"])
+	@script(gestures=[
+		"kb:enter",
+		"kb:numpadEnter",
+		"kb:tab",
+		"kb:control+c",
```

Comment:
How could we make sure that this list of gestures is complete?

Reply:
It probably isn't unfortunately.